packages/inferno-from-react-generator/src/expressions/imports-aggregation.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  Decorator,
+  GeneratorContext,
+  ImportClause,
+  ImportDeclaration,
+  StringLiteral,
+  VariableExpression,
+} from '@devextreme-generator/core';
+
+type AgregateableImportDeclaration = Pick<ImportDeclaration, 'toString'>;
+
+export class ImportsAggregation {
+  private declarations: AgregateableImportDeclaration[];
+
+  public decorators!: Decorator[];
+
+  public modifiers!: string[];
+
+  public importClause!: ImportClause;
+
+  public moduleSpecifier!: StringLiteral;
+
+  public context!: GeneratorContext;
+
+  constructor(...declarations: AgregateableImportDeclaration[]) {
+    this.declarations = declarations;
+  }
+
+  toString(): string {
+    return this.declarations.map((d) => d.toString()).join('\n');
+  }
+
+  replaceSpecifier() {
```

Comment:
Question.

Maybe I missed something.
But, why do we need these not implemented methods in this class?

Reply:
I've made this in order to fit ImportDeclaration type and avoid implementing these methods. implementation is not required and even more - not obvious. E.g. what should the `add` method regarding multiple import clauses?

Whatever, @MikeVitik  asked me to rewrite this part, take a look at the new one